solutions/3690/01.py

```
@@ -0,0 +1,35 @@
+class Solution:
+    def minSplitMerge(self, nums1: List[int], nums2: List[int]) -> int:
```

Ruff (0.14.8)

Undefined name `List`

(F821)